### ISSUE TITLE
cannot reproduce your results using pre-trained model

### ISSUE THREAD

**Issue by tsing90** [huochaitiantang/pytorch-deep-image-matting#8]
my fault ... it worked perfect ! closing the issue with my output .............

![1](https://user-images.githubusercontent.com/20683656/54197837-e7805b00-44bc-11e9-80fe-97d7d18d81e4.png)

**Comment by FantasyJXF** [comment]
@tsing90  Hi, could you please share how you get this output?  I download the Adobe dataset too, and I still don't know why the fg image are not original image, which all have some shadow on the outside of body.

And another question is did you test the model on your own images. I tried to segment the object with instance segmentation and add trimap to the mask, then use THE author's pretrained model, not so good in my case, maybe I didn't operate right.

Here is my result of your test, outsied boader ....
![boy-1518482_1920](https://user-images.githubusercontent.com/16235526/61098446-99db2580-a491-11e9-818c-fc76a5a8445b.png)
